runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -58,51 +58,57 @@
 	private final ItemPricesConfig config;
 	private final TooltipManager tooltipManager;
 	private final StringBuilder itemStringBuilder = new StringBuilder();
+	private final ItemPricesPlugin plugin;
 
 	@Inject
-	ItemManager itemManager;
+	private ItemManager itemManager;
 
 	@Inject
-	ItemPricesOverlay(Client client, ItemPricesConfig config, TooltipManager tooltipManager)
+	ItemPricesOverlay(Client client, ItemPricesConfig config, TooltipManager tooltipManager, ItemPricesPlugin plugin)
 	{
 		setPosition(OverlayPosition.DYNAMIC);
 		this.client = client;
 		this.config = config;
+		this.plugin = plugin;
 		this.tooltipManager = tooltipManager;
 	}
 
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (client.isMenuOpen())
+		if (plugin.showPrices || !config.showOnALT())
```

Comment:
Could this be organized the other way around? e.g. `if (config.showOnAlt() && !plugin.altPressed) return;` or something? The logic was a bit confusing here initially reading it.

Reply:
Changed the names to be more clear and added a comment